C_CPP/Fundamentals/Functions & Recursion and its types/Audio_passing_parameters_to_a_function.md

```diff
@@ -0,0 +1,12 @@
+# PASSING PARAMETERS TO A FUNCTION
+
+*In this audio file,I have covered following topics:-*
+1. What is calling function and called function?
+2. What is the difference between a calling function and a called fucntion?
+3. How we can pass arguments from calling function to called function?In how many ways?
+4. What is call by value?How it is used while passing paramters to a function?What are its advantages and disadvantages?
+5. What is call by reference?Why and in which condition we use call by reference instead of using call by value method?
+6. Some examples based on both call by value and call by reference.
+
+## Audio file
+[Audio for passing parameters to a function](https://drive.google.com/file/d/1NAatKPELWlApVAOIkjzf-FwkwBbdTKZn/view?usp=sharing)
```

Comment:
@Harsh652-cpu dictating the code in your audio is not a good practice, please mention only the major syntax necessary and explain the rest in words

Reply:
Done @apeksha235